docs/features/software-catalog/installation.md

```diff
@@ -112,59 +113,12 @@ cd packages/backend
 yarn add @backstage/plugin-catalog-backend
 ```
 
-Make sure the version of `@backstage/plugin-catalog-backend` matches the version
-of other `@backstage` packages. You can update it in
-`packages/backend/package.json` if it doesn't.
-
 ### Adding the Plugin to your `packages/backend`
 
 You'll need to add the plugin to the `backend`'s router. You can do this by
-creating a file called `packages/backend/src/plugins/catalog.ts` with the
-following contents to get you up and running quickly.
-
-```ts
-import {
-  createRouter,
-  DatabaseEntitiesCatalog,
-  DatabaseLocationsCatalog,
-  DatabaseManager,
-  HigherOrderOperations,
-  LocationReaders,
-  runPeriodically,
-} from '@backstage/plugin-catalog-backend';
-import { PluginEnvironment } from '../types';
-import { useHotCleanup } from '@backstage/backend-common';
-
-export default async function createPlugin({
-  logger,
-  database,
-}: PluginEnvironment) {
-  const locationReader = new LocationReaders(logger);
-
-  const db = await DatabaseManager.createDatabase(database, { logger });
-  const entitiesCatalog = new DatabaseEntitiesCatalog(db);
-  const locationsCatalog = new DatabaseLocationsCatalog(db);
-  const higherOrderOperation = new HigherOrderOperations(
-    entitiesCatalog,
-    locationsCatalog,
-    locationReader,
-    db,
-    logger,
-  );
-
-  useHotCleanup(
-    module,
-    runPeriodically(() => higherOrderOperation.refreshAllLocations(), 10000),
-  );
-
-  return await createRouter({
-    entitiesCatalog,
-    locationsCatalog,
-    higherOrderOperation,
-    logger,
-  });
-}
-```
+creating a file called `packages/backend/src/plugins/catalog.ts` with contents
+matching
+[catalog.ts in the create-app template](https://github.com/backstage/backstage/blob/master/packages/create-app/templates/default-app/packages/backend/src/plugins/catalog.ts).
```

Comment:
This is a sizable code block and had drifted out of date, linking to the source of truth here seemed better.